.github/workflows/ci.yml

```diff
@@ -41,6 +41,13 @@ jobs:
           - { name: 'iOS x86_64',         target: x86_64-apple-ios,         os: macos-latest,    }
           - { name: 'iOS Aarch64',        target: aarch64-apple-ios,        os: macos-latest,    }
           - { name: 'web',                target: wasm32-unknown-unknown,   os: ubuntu-latest,   }
+        exclude:
+          # Android is tested on stable-3
+          - toolchain: '1.65.0'
+            platform: { name: 'Android', target: aarch64-linux-android, os: ubuntu-latest, options: '--package=winit --features=android-native-activity', cmd: 'apk --' }
+        include:
+          - toolchain: '1.69.0'
+            platform: { name: 'Android', target: aarch64-linux-android, os: ubuntu-latest, options: '--package=winit --features=android-native-activity', cmd: 'apk --' }
```

Comment:
If you want to special case android like that, you must tell cargo to ignore `rust_version` in the `Cargo.toml`, otherwise it won't build iirc even when version is good enough. The flag is `--ignore-rust-version`.

Reply:
Shouldn't we just change the `rust-version` field to the Android MSRV? Still useful I would argue.

Reply:
Ah wait, that would prevent lower version from working as well ...
Yeah, I'm in favor of removing it then.

Reply:
Actually no ... sorry, it's still useful, just a lower bound is better then nothing.
Apologies, I got confused here.

Reply:
I agree with @daxpedda, as it causes a compile error `rust-version` serves best as a lower bound for the MSRV.

Reply:
Can we have a `rust-version` per target and not global? I think it's global, no? That's why I've suggested to sort of dynamically add it into the `toml` or maybe somehow fix the version....

Reply:
It's always fine to build with a newer version of Rust, and there's no need to use `--ignore-rust-version`.

`--ignore-rust-version` would be applicable for the opposite situation where you know that Winit technically can be built with an even older Rust version than is specified in the `rust-version`.

That said though - removing the rust-version sounds good to me and Winit can instead just rely on CI to check the MSRV because that gives more flexibility to test different versions for different platforms.

Cargo will anyway calculate the real minimum rust-version based on the minimum rust-version across the dependencies selected for a specific platform at build time.

Reply:
The thing about `rust-version` is that some crate might force MSRV larger than it can be built with. I guess I just forgot how `rust-version` works exactly, and if you can build just fine when there's incompat between rust versions, then it sounds fine the way it's now.